editor/src/components/editor/actions/action-creators.ts

```diff
@@ -0,0 +1,1208 @@
+import { LayoutSystem } from 'utopia-api' // TODO fixme this imports utopia-api
```

Comment:
Why is this an issue?

Reply:
since this file imports almost nothing else, I felt bad for having to pull in utopia-api (and thus emotion, and other such stuff we are using there). probably not a massive problem, but since we only use it to get an Enum (which are soft deprecated anyways), it wouldn't be hard to switch that type to a string literal union, and then we wouldn't need to import it, we could just `import type` it